aws-wafv2-loggingconfiguration/aws-wafv2-loggingconfiguration.json

```diff
@@ -111,7 +111,7 @@
 									"minItems": 1,
 									"items": {
 										"type": "string",
-										"pattern": "^[\\/]+([^~]*(~[01])*)*{1,512}$"
+										"pattern": "^[\\/]+([^~]*(~[01])*){1,512}$"
```

Comment:
Per the docs `([/])|([/](([^~])|(~[01]))+)` https://docs.aws.amazon.com/waf/latest/APIReference/API_JsonMatchPattern.html

I think this is the right fix.